editing/fx/passes/eps.py

```diff
@@ -424,6 +439,10 @@ def run_pass(self, gm : fx.GraphModule):
                             print("[AnnotateEpsPass] Mismatching input signedness in node with no signedness propagation function! signedness propagation will likely be wrong!")
                             print(f"                    -> Node: {node.name}, Key: : {k}, signed_in: {node_in_signed}")
                             if (self.verbose): print(f"[AnnotateEpsPass] Using identity signed propagation on node with op {node.op}, target {node.target}!")
+
+                        if node.op == "call_function" and node.target == torch.cat:
+                            node_in_signed = [inp.meta['quant'].signed_out for inp in node.args[0]]
```

Comment:
Please extract this into a more proper format - maybe a function that returns node_in_signed at the top of this code block rather than hacking this in here.

Reply:
Addressed in [this commit.](https://github.com/pulp-platform/quantlib/pull/15/commits/09d2983fd33738ad3b193fa3917588587c075cad)